transformers/modeling_seq2seq.py

```diff
@@ -28,14 +28,15 @@
 logger = logging.getLogger(__name__)
 
 
-class PreTrainedSeq2seq(nn.Module):
+class PreTrainedSeq2seq(PreTrainedModel):
```

Comment:
Why should we depend on `PreTrainedModel`?

Reply:
yeah, no reason since we redefine `from_pretrained`.